src/platforms/common/configuration/releases.mdx

```diff
@@ -43,21 +43,13 @@ This tags each event with the release value. We recommend that you tell Sentry a
 
 After configuring your SDK, you can install a repository integration or manually supply Sentry with your own commit metadata. Read our documentation about [Releases](/product/releases/) for further information about integrations, associating commits, and telling Sentry when deploying releases.
 
-<PlatformSection supported={["apple"]}>
+<PlatformSection supported={["apple", "android", "javascript"]}>
```

Comment:
should we add `flutter`? react-native also has support for release health

Reply:
Good catch!